src/app/github-link-area/examples/github-link-area-example.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="padding-15">
+  <div class="row padding-bottom-15">
+    <div class="col-xs-12">
+      <h4>GitHubLink Component Example</h4>
+      <hr/>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-sm-12">
+      <div class="form-group">
+        <github-link-area [content]="content"></github-link-area>
+      </div>
+    </div>
+  </div>
+  <div class="row padding-bottom-15 padding-top-15">
+    <div class="col-xs-12">
+      <h4>Component Options</h4>
+      <hr/>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-sm-12 sample-form">
+      <form class="form-horizontal">
+        <div class="form-group">
+          <label class="control-label" for="content">Content: </label><br>
```

Comment:
`for` should match the `id` of the `textarea` not the `name`

Reply:
Fixed.